main.go

```diff
@@ -41,9 +41,10 @@ func main() {
 	/* Github */
 	funcMap["recentContributions"] = recentContributions
 	funcMap["recentPullRequests"] = recentPullRequests
-	funcMap["recentRepos"] = recentRepos
 	funcMap["recentForks"] = recentForks
 	funcMap["recentReleases"] = recentReleases
+	funcMap["recentRepos"] = recentRepos
+	funcMap["orgPopularRepos"] = orgPopularRepos
```

Comment:
This is not specific for organizations though, you should be able to use it for users as well. I would rename it to `popularRepos`

Reply:
Or `userPopularRepos`

Reply:
Ah hold on, I finally figured out how to make this work for users and orgs 🎊  I'll update the query + solution across the board here. Thanks for the nudge :)